### ISSUE TITLE
dfyn.network does not work

### ISSUE THREAD

**Issue by tonightlife** [vfat-tools/vfat-tools#1045]
Hey there, 
great work guys!! Love the tool.
But I tried to connect to dfyn.network and it does not work.
I got different error.
The last one is:

{
  "code": -32603,
  "message": "Internal JSON-RPC error.",
  "data": {
    "details": "Internal RPC Error. Try again after some time..",
    "code": -32005
  }
}

Last time I got an error something like: to many request...

Please help.

thanks a lot and best regards
Matt

**Comment by vfat0** [comment]
Please see this https://www.reddit.com/r/DFYN_Network/comments/o8whhy/step_by_step_guide_how_to_use_vfat_tools_to/

**Comment by tonightlife** [comment]
Hey, thanks. 
I've tried several RPC urls, but still RPC errors:

Reading smart contracts...

{
  "code": -32603,
  "message": "Internal JSON-RPC error.",
  "data": {
    "code": -32000,
    "message": "header not found"
  }
}

Maybe it is the firefox? Shall I try another browser?

**Comment by tonightlife** [comment]
Now I got with https://matic-mainnet-archive-rpc.bwarelabs.com
this error

{
  "code": -32603,
  "message": "NetworkError when attempting to fetch resource.",
  "data": {
    "originalError": {}
  }
}
Oops something went wrong. Try refreshing the page.

**Comment by vfat0** [comment]
It is almost definitely the RPC url, the page works ok for me. That last one is probably offline. Did you try regisering for a private RPC? https://docs.matic.network/docs/develop/network-details/network/

**Comment by tonightlife** [comment]
No I did not, but I just solved the problem/issue.
I think it was the security settings of the firefox. See image before and after.
![before](https://user-images.githubusercontent.com/8170294/125046314-3004af00-e09e-11eb-9844-598ee3441277.jpg)
![after](https://user-images.githubusercontent.com/8170294/125046319-3135dc00-e09e-11eb-999b-e31db4b110eb.jpg)

Maybe it is a good idea to add this info to the doc?
Looking forward what happens next with this project, I think about to contribute soon.
Thanks a lot for your help, I appreciate that!
